src/questions/introduction_to_defi.yaml

```diff
@@ -0,0 +1,1147 @@
+    - content: Opaque
+      key: D
+      order: 3
+- uuid: 8f3f5fc2-869b-4040-9363-c95b49ed8a92
+  type: SingleChoice
+  content: In what form does DeFi transfer take place?
+  hint: NoHint
+  explanation: DeFi transfer takes place in the peer to peer form.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Computer to computer
+      key: A
+      order: 0
+    - content: Peer to Peer
+      key: B
+      order: 1
+    - content: Product to Product
+      key: C
+      order: 2
+    - content: None of these
+      key: D
+      order: 3
+- uuid: 33f9f434-48ae-4940-9ee7-5dae700e3fc1
+  type: MultipleChoice
+  content: What are the inefficiencies of a centralised finance system? 
+  hint: Disadvantages of CeFi
+  explanation: Banks charge 3% interest from their users on each credit card swipe and it can take upto 2 days to settle a stock transaction.
+  answerKeys:
+    - A
+    - B
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Bank charges 3 % for credit card swipes.
+      key: A
+      order: 0
+    - content: It takes 2 days to settle a stock transaction
+      key: B
+      order: 1
+    - content: Banks provide safe vaults for keeping valuable assets.
+      key: C
+      order: 2
+    - content: Banks provide control over the printing of currency to prevent inflation.
+      key: D
+      order: 3
+- uuid: c01bebee-a35c-4435-b622-7688f80ce807
+  type: SingleChoice
+  content: What percentage of the world population is unbanked?
+  hint: NoHint
+  explanation: Around 30% of the world poupulation is underbanked and do not have access to necessary bank facilities.
+  answerKeys:
+    - B
+  subTopics:
+    - "centralised-vs-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: 5%
+      key: A
+      order: 0
+    - content: 30%
+      key: B
+      order: 1
+    - content: 10%
+      key: C
+      order: 2
+    - content: 25%
+      key: D
+      order: 3
+- uuid: 9f864936-5494-4771-bf6e-c744a9f18618
+  type: SingleChoice
+  content: What do you understand by Peer-to-peer lending?
+  hint: NoHint
+  explanation: P2P or Peer to Peer lending refers to transfer of funds from one wallet to another without a third party observation.
+  answerKeys:
+    - D
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: One bank account to another transaction.
+      key: A
+      order: 0
+    - content: One crypto wallet to another under a third party observation.
+      key: B
+      order: 1
+    - content: One email address to another.
+      key: C
+      order: 2
+    - content: One crypto wallet to another without a third party observation.
+      key: D
+      order: 3
+- uuid: 93e24b6c-9a74-4bae-8c93-c439620fe67e
+  type: SingleChoice
+  content: What is the Total value locked in DeFi as of November 2021?
```

Comment:
`What is the Total value` -> `What was the Total value`

Also, since the TLV changes a lot, not sure if it will be easy for user.

We can also ask what was the `peak` TLV in 2021

Reply:
Done